src/Razor/src/Microsoft.CodeAnalysis.Remote.Razor/Initialization/RemoteLanguageServerFeatureOptions.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.Composition;
+using Microsoft.CodeAnalysis.Razor.Remote;
+using Microsoft.CodeAnalysis.Razor.Workspaces;
+
+namespace Microsoft.CodeAnalysis.Remote.Razor;
+
+[Shared]
+[Export(typeof(LanguageServerFeatureOptions))]
+internal class RemoteLanguageServerFeatureOptions : LanguageServerFeatureOptions
+{
+    // It's okay to use default here because we expect the options to be set before the first real OOP call
+    private static RemoteClientInitializationOptions s_options = default;
```

Comment:
To my mind this shouldn't need to be static, because the service is shared, and so we should be able to get an instance from the export provider, update it, and then a future request that gets an instance from the export provider should get the same instance. That isn't happening, and I'm guessing its a bit of servicehub oddness (I don't think it's how we're using MEF because [I tested](https://gist.github.com/davidwengier/87417440b6c51d71a88455ac83ba3f36), but I decided not to dwell on it in this case. We can always change this in future when there is more time to dig in to things.